scripts/vespa_local/vespa_local.py

```diff
@@ -587,6 +599,15 @@ def deploy_config(args):
     os.system(f'vespa deploy "{here}"')
 
 
+def generate_application_package(args):
+    # Create instance of VespaLocal
+    # vespa_local_instance is used for starting vespa & generating application package.
+    vespa_local_instance = VespaLocalSingleNode()
```

Comment:
Instead of hardcoding it to be `VespaLocalSingleNode()`, can we use the `--Shards` and `--Replicas` argument here so it's dynamic as well? Might be confusing if some of the functions are hardcoded to single node and some are not.